### PR TITLE
64 expand wiki

### LINKED ISSUE #64: Expand wiki
- [x] add workflows
- [ ] add rationale
- [ ] ...

### PR DESCRIPTION
**Pull request by soriadelva**

### CODE REVIEW

File: docs/aspbo_flows-Dataflows_overview.drawio.png
Comment (SanderDevisscher) [inline docs/aspbo_flows-Dataflows_overview.drawio.png:1]: @soriadelva not to nitpick but the databases/buckets are not allways the same color

File: docs/aspbo_flows-Vespa velutina management.drawio.png
Comment (SanderDevisscher) [inline docs/aspbo_flows-Vespa velutina management.drawio.png:1]: Different db/bucket color ?

**Review by SanderDevisscher — APPROVED** [review]
Some small remarks, you are free to ignore these. Ifnot just reupload the fixed png and merge.

File: docs/aspbo_flows-Dataflows_overview.drawio.png
Comment (soriadelva) [reply docs/aspbo_flows-Dataflows_overview.drawio.png:1]: ah indeed! it's the PROD bucket in the upload_files pane, right? I'll fix it!

Comment (SanderDevisscher) [reply docs/aspbo_flows-Dataflows_overview.drawio.png:1]: ah, you differentiate the external db's (gbif/iasset) from the internal db (UAT/PRD) and you differentiate between the development (UAT) and production (PRD) colorwise ? Ifso I'm wrong you used the correct colors!

**Pull request merged**